packages/page-accounts/src/Accounts/modals/Qr.tsx

```diff
@@ -101,6 +127,8 @@ function QrModal ({ className = '', onClose, onStatusChange }: Props): React.Rea
                   <p>{t<string>('The local name for this account. Changing this does not affect your on-line identity, so this is only used to indicate the name of the account locally.')}</p>
                 </Modal.Column>
               </Modal.Columns>
+              {!isAddress && <PasswordInput onEnter={_onSave}
+                passwordStates={[passwordStateHooks, password2StateHooks]}/>}
```

Comment:
```suggestion
              {!isAddress && (
                <PasswordInput
                  onEnter={_onSave}
                  passwordStates={[passwordStateHooks, password2StateHooks]}
                />
              )}
```

Here really would prefer to no create a new object each render for passwordStates. Depending on what we need rather put the hooks inside the component itself and have a onChange that sends a single back to the parent. (When all is valid)